sys/nginx/nginx.conf

```diff
@@ -11,6 +13,29 @@ server {
 
     root /application/current/public;
 
+    set $skip_cache 0;
+    if ($request_method = POST) { set $skip_cache 1; }
+    if ($query_string != "") { set $skip_cache 1; }
+    if ($request_uri ~ "^/$") { set $skip_cache 1; }
+
+    fastcgi_cache cached_badges;
+    fastcgi_cache_bypass $skip_cache;
+    fastcgi_cache_background_update on;
+    fastcgi_cache_key "$scheme$request_method$host$request_uri";
+    fastcgi_cache_use_stale error timeout invalid_header http_500 http_503;
+    fastcgi_cache_valid 200 1m;
```

Comment:
I don't get why the TTL is not applied and it stays cached for longer. Some help here is needed.
As a workaround I've put in place an horrible cronjob which removed the files from cache folder.

Reply:
I can't get my head around this, it seems I'm not the only one forcing cleaning manually the nginx cache. I don't believe it, or maybe it's only working for the commercial version...

Ref:
 - https://www.jverdeyen.be/devops/nginx-clear-fastcgi-cache/
 - https://stackoverflow.com/questions/14145924/how-do-i-clear-fastcgi-cache-with-php
 - https://www.sorendam.com/clear-nginx-cache/